image/templates/helm/stackrox-secured-cluster/internal/defaults/30-base-config.yaml.htpl

```diff
@@ -100,7 +100,7 @@ admissionControl:
             labelSelector:
               matchLabels:
                 app: admission-control
-
+  namespaceSelector: null
```

Comment:
I think this change can be removed. We are not setting fields to `null` here, this is already taken care of by adding to the config shape.

Reply:
makes sense, i removed it 😄